src/AccessibilityInsights.SharedUx/Settings/AppLayout.cs

```diff
@@ -91,7 +91,8 @@ public void LoadLayoutIfPrevVersion(double top, double left)
             if (this.Version == null || string.CompareOrdinal(this.Version, AppLayout.CurrentVersion) != 0)
             {
                 // for now we enforce the layout to be always be default if the version mismatch.
-                // it will reduce the unnecessary crashes or noize. but it may lose the previous layout info.
+                // it will reduce the unnecessary crashes or
+                // noise. but it may lose the previous layout info.
```

Comment:
Keep original line breaks, use comma
```suggestion
                // it will reduce the unnecessary crashes or noise, but it may lose the previous layout info.
```